call_for_proposal.html

```diff
@@ -0,0 +1,284 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<meta name="description" content="">
+	<meta name="author" content="">
+
+	<link rel="icon" type="image/ico" href="https://a0.awsstatic.com/main/images/site/fav/favicon.ico" />
+	<link rel="shortcut icon" type="image/ico" href="https://a0.awsstatic.com/main/images/site/fav/favicon.ico" />
+	<link rel="apple-touch-icon" sizes="57x57" href="https://a0.awsstatic.com/main/images/site/touch-icon-iphone-114-smile.png"
+	/>
+	<link rel="apple-touch-icon" sizes="72x72" href="https://a0.awsstatic.com/main/images/site/touch-icon-ipad-144-smile.png"
+	/>
+	<link rel="apple-touch-icon" sizes="114x114" href="https://a0.awsstatic.com/main/images/site/touch-icon-iphone-114-smile.png"
+	/>
+	<link rel="apple-touch-icon" sizes="144x144" href="https://a0.awsstatic.com/main/images/site/touch-icon-ipad-144-smile.png"
+	/>
+
+	<link rel="canonical" href="https://communityday.awsugblr.in/" />
+
+	<meta property="og:url" content="https://communityday.awsugblr.in" />
+	<meta property="og:type" content="website" />
+	<meta property="og:title" content="AWS Community Day 2018 " />
+  <meta property="og:description" content="AWS Community Day is a community-organized event featuring technical discussions led by expert AWS users and community leaders. AWS Community Day is being organized by AWS Users Group - Bangalore. This group is for anyone interested in Amazon Web Services, Cloud Computing, AWS design implementation and servicing, EC2, S3, High Scalability Computing, Real Life uses of AWS and cloud servers, SaaS, IAAS, PAAS and many more" />
+	<meta property="og:image" content="https://s3.ap-south-1.amazonaws.com/aws-community-blr/community-day-logo-sq.png" />
+	<meta property="og:image:secure_url" content="https://s3.ap-south-1.amazonaws.com/aws-community-blr/community-day-logo-sq.png"
+	/>
+	<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+	<title>AWS Community Day 2018 Bengaluru - Call for Proposal</title>
```

Comment:
please change the title to "Call for Proposals - AWS Community Day 2018 Bengaluru"